satellite/satellitedb/attribution.go

```diff
@@ -24,6 +25,9 @@ func (keys *attributionDB) Get(ctx context.Context, projectID uuid.UUID, bucketN
 		dbx.ValueAttribution_ProjectId(projectID[:]),
 		dbx.ValueAttribution_BucketName(bucketName),
 	)
+	if err == sql.ErrNoRows {
+		return nil, attribution.ErrBucketNameNotFound.New(string(bucketName))
```

Comment:
its not that bucket is not found, it means that there are no attribution for such bucket name.

Reply:
updated the error to ErrNoAttributionFound = errs.Class("bucket not attributed").